utility-js/Utility.js

```diff
@@ -61,22 +58,6 @@ class Utility {
     return this._householdExists(hhAddress) ? this.households[hhAddress] : {};
   }
 
-  /**
```

Comment:
the api endpoint` /network `in `ned-server/index.js` has to be changed as well, as it calls these getters

Reply:
good catch! fixed in a91bea6.

Reply:
Good catch! Fixed in a91bea6.